firebase-perf/src/main/java/com/google/firebase/perf/application/FragmentStateMonitor.java

```diff
@@ -66,11 +66,10 @@ public void onFragmentResumed(@NonNull FragmentManager fm, @NonNull Fragment f)
         new Trace(getFragmentScreenTraceName(f), transportManager, clock, appStateMonitor);
     fragmentTrace.start();
 
-    if (f.getParentFragment() != null) {
-      fragmentTrace.putAttribute(
-          Constants.PARENT_FRAGMENT_ATTRIBUTE_KEY,
-          f.getParentFragment().getClass().getSimpleName());
-    }
+    Fragment parent = f.getParentFragment();
+    fragmentTrace.putAttribute(
+        Constants.PARENT_FRAGMENT_ATTRIBUTE_KEY,
+        parent == null ? Constants.NO_PARENT_FRAGMENT : parent.getClass().getSimpleName());
```

Comment:
Could you add a test to check the parent_fragment attribute value when there is no parent fragment?

Reply:
Sounds good, added `fragmentTraceCreation_addsSpecialAttributeValue_whenFragmentDoesNotHaveParent` in `FragmentStateMonitorTest`